connectors/services/base.py

```diff
@@ -39,9 +39,6 @@ async def run(self):
         self.running = True
         try:
             await self._run()
-        except Exception as e:
```

Comment:
I think this still makes sense?

As a derived class of `BaseService`, you can implement `_run` without any `try...except...`.

Reply:
I think it's okay?

I don't see why BaseService should not re-raise the error if it happens in `self._run`

Reply:
Got it. Thanks for the explanation